animations-baker.js

```diff
@@ -260,6 +270,22 @@ const {CharsetEncoder} = require('three/examples/js/libs/mmdparser.js');
 
       animations.push(animation);
     }
+
+    const swordStrikeContinuousAnimation = animations.filter(a => a.name === 'sword_strike_continuous.fbx')[0];
+    let animation;
+    animation = trimClip(swordStrikeContinuousAnimation.clone(), 0, 20 / 30);
```

Comment:
This is really not good to depend on. We made these animations. Why is this not fixed on the art side?

Reply:
Used pre-splitted attack animations by textametron, instead of `trimClip()`. https://github.com/webaverse/app/pull/2804/commits/2b5c1c4c12ea016b5b4e2d4e0b15a7147e17ed3e